### ISSUE TITLE
Add ylim logic to plot.netdx for plots without sims or qnts  

### ISSUE THREAD

**Issue by sgoodreau** [EpiModel/EpiModel#676]
It's reasonable to expect a user may want a plot with just the mean line and the target.  This currently works, but gives a not-very-useful warning:

```
rm(list=ls())
mynet <- network_initialize(1000)
myform <- ~edges+degree(0)
mystats <- c(1000, 200)
mydiss <- ~offset(edges)
mydurs <- c(30)
myest <- netest(mynet, myform, mystats, dissolution_coefs(mydiss,mydurs))
mydx <- netdx(myest, nsteps=100, nsims=5, keep.tnetwork = TRUE)
```

```
> plot(mydx, type='formation', sim.lines=F, qnts=F)
Warning messages:
1: In min(qnt.min, na.rm = TRUE) :
  no non-missing arguments to min; returning Inf
2: In max(qnt.max, na.rm = TRUE) :
  no non-missing arguments to max; returning -Inf

> plot(mydx, type='duration', sim.lines=F, qnts=F)
1: In min(qnt.min, na.rm = TRUE) :
  no non-missing arguments to min; returning Inf
2: In max(qnt.max, na.rm = TRUE) :
  no non-missing arguments to max; returning -Inf

> plot(mydx, type='dissolution', sim.lines=F, qnts=F)
1: In min(qnt.min, na.rm = TRUE) :
  no non-missing arguments to min; returning Inf
2: In max(qnt.max, na.rm = TRUE) :
  no non-missing arguments to max; returning -Inf
```

Also: check to see if there are other argument combinations that produce this.
Also: check to see if this occurs in other plotting functions (e.g. `plot.netsim`).

**Comment by smjenness** [comment]
@sgoodreau Thanks for the report. This is definitely a bug to be fixed.

**Comment by smjenness** [comment]
Closed by #702
